requirements.txt

```diff
@@ -1 +1 @@
-pymongo==3.12.3
+pymongo>=4.2
```

Comment:
Why did you decide to use `>=` over pinning a particular version of PyMongo via `==`? What's the reasoning behind this change?

Reply:
dev artifact - just used it to pull the latest for some testing. Pinned to 4.2.0 now, thanks!

Reply:
Ok. I suggest we never do this. Even if you want to test against the latest released version of PyMongo, it would be better to pin that version explicitly. Using `>=` in requirements.txt can cause a whole lot of confusion if a new version of a dependency is released (the schedule of which is besides our control).

Overall though, I don't think testing against the latest version makes sense in a PR like this. Upgrades should generally be performed with the least risk and thus without jumping multiple versions at once. If your goal is to upgrade PyMongo to v4.2, then testing against higher versions is just gonna create red herrings.